packages/terra-demographics-banner/tests/nightwatch/demographics-banner-spec.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-unused-expressions */
+
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  afterEach: (browser, done) => {
```

Comment:
Fixed it here. https://github.com/cerner/terra-core/pull/191/commits/b94f4efae5adff93b8c7fcf4356ced63480f3c47